.eslintrc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "standard",
+  "extends": ["standard", "plugin:prettier/recommended"],
```

Comment:
It would be nice to introduce prettier in a separate PR.

Reply:
done